### PR TITLE
[Fix] Invidious Engine

### PR DESCRIPTION
**Pull request by mrwormo**
## What does this PR do?

Fixes Invidious engine by enabling requests by randomly picking amongst working instances declared in settings.yml.

## Why is this change important?

To get a working invidious engine.

## How to test this PR locally?

Make run

## Related issues

#2161

### CODE REVIEW

File: searx/settings.yml [hunk 1]
```diff
@@ -478,7 +478,9 @@ engines:
 
   - name : invidious
     engine : invidious
-    base_url : 'https://invidio.us/'
+    base_url :
```
Comment (kvch) [inline searx/settings.yml]: There is an extra space after `:`. Please remove it.

**Review by kvch — CHANGES_REQUESTED** [review]
Small formatting request.

**Comment by kvch** [comment]
I have tested the engine, and everything seems to work. Once the extra space is removed, it is ready to be merged.

File: searx/settings.yml (hunk 1 above)
Comment (mrwormo) [reply searx/settings.yml]: Hello,
I did it.
Thx.

**Comment by unixfox** [comment]
@mrwormo Can you squash all the commits into one?

**Comment by return42** [comment]
> [fix] Invidious engine by enabling requests by randomly picking amongst working instances

If you squash, can you use this as commit message / thanks!

**Comment by return42** [comment]
Prepared squash commit dbb19e511e you can pull from  https://github.com/return42/searx/commits/invidious-engine

**Review by dalf — APPROVED** [review]
Thank you!

**Comment by mrwormo** [comment]
Hey @dalf,
The second attempt was the good one :wink:
It means issue #2161 can be closed.
Thanks peeps for your patience and kindness...

**Pull request merged**